### ISSUE TITLE
Can't install on Windows

### ISSUE THREAD

**Issue by tr4g** [javascript-obfuscator/javascript-obfuscator#783]
So I'm trying to install on Windows, but when it reaches the opencollective part, it crashes because it tries to use "||" operator which doesn't exist in Powershell:

```
En línea: 1 Carácter: 16
+ opencollective || exit 0
+                ~~
El token '||' no es un separador de instrucciones válido en esta versión.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : InvalidEndOfLine
```
Translation: "Token `||` is not a valid instruction separator on this version"

**Comment by da411d** [comment]
Maybe try usual command line?

**Comment by sanex3339** [comment]
Agree. Use regular cmd on windows.

**Comment by sanex3339** [comment]
Will be released with the next release.

**Comment by sanex3339** [comment]
Released as `2.6.2`
